ecl/hql/hqlparse.cpp

```diff
@@ -604,7 +605,24 @@ void HqlLex::checkSignature()
             }
         }
         else
+        {
             yyParser->inSignedModule = true;
+            yyParser->gpgSignature.clear();
+            const char sigprefix[] = "Good signature from \"";
```

Comment:
const char * is more normal.  Although sizeof(x)-1 is theoretically slightly faster than strlen(x) I don't think it is worth it 
a) because it is less obvious and a bit of a trick and b) gcc constant folds strlen(constant string) so it isn't any more efficient.